### PR TITLE
Update Elasticsearch version

### PR DESCRIPTION
**Pull request by lithrel**
- Update Elasticsearch version to prod version 7

### CODE REVIEW

**Comment by sagarsdeshmukh** [comment]
Hi Flo, While running an elastic hq command, it stops with below error -
```
sagardeshmukh@Sagars-Mac-mini planet4-develop % npm run elastic:hq       

> planet4-develop@0.12.0 elastic:hq
> node ./scripts/elastic.js hq

✔  (in 0s 903ms)
WARN[0000] /Users/sagardeshmukh/.wp-env/70fac6f4b14bb9d08a9915bb4117bb3f/docker-compose.yml: `version` is obsolete 
WARN[0000] /Users/sagardeshmukh/Desktop/Sagar Doc/planet4-develop/scripts/docker-compose.p4.yml: `version` is obsolete 
[+] Running 11/11
 ✔ elastichq Pulled                                                                                                                                                                                                                                                     32.5s 
   ✔ 48ecbb6b270e Pull complete                                                                                                                                                                                                                                          1.6s 
   ✔ 692f29ee68fa Pull complete                                                                                                                                                                                                                                          0.8s 
   ✔ f75fc7ac1098 Pull complete                                                                                                                                                                                                                                          7.8s 
   ✔ c30e40bb471c Pull complete                                                                                                                                                                                                                                          2.0s 
   ✔ 51a8cc25b36b Pull complete                                                                                                                                                                                                                                          3.2s 
   ✔ 0b8b8dd240d2 Pull complete                                                                                                                                                                                                                                         26.1s 
   ✔ 87aaa07ce2a1 Pull complete                                                                                                                                                                                                                                          7.3s 
   ✔ bf0fc67433bd Pull complete                                                                                                                                                                                                                                         17.8s 
   ✔ edf4a873263c Pull complete                                                                                                                                                                                                                                          8.7s 
   ✔ 4014c9eef65e Pull complete                                                                                                                                                                                                                                         12.6s 
[+] Running 2/1
 ✔ Container 70fac6f4b14bb9d08a9915bb4117bb3f-elastichq-1                                                                                                   Created                                                                                                      0.2s 
 ! elastichq The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested                                                                                                              0.0s 
Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:5000 -> 0.0.0.0:0: listen tcp 0.0.0.0:5000: bind: address already in use
node:child_process:935
    throw err;
    ^

Error: Command failed: docker compose -f $(npx wp-env install-path)/docker-compose.yml -f scripts/docker-compose.p4.yml up --force-recreate -d elastichq
    at checkExecSyncError (node:child_process:861:11)
    at execSync (node:child_process:932:15)
    at run (/Users/sagardeshmukh/Desktop/Sagar Doc/planet4-develop/scripts/lib/run.js:15:10)
    at Object.<anonymous> (/Users/sagardeshmukh/Desktop/Sagar Doc/planet4-develop/scripts/elastic.js:35:3)
    at Module._compile (node:internal/modules/cjs/loader:1198:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 6732,
  stdout: null,
  stderr: null
}
```

**Comment by lithrel** [comment]
> The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

@sagarsdeshmukh  Right, the elastichq image is far too old, and elastichq itself is not maintained anymore. I removed these, we'll have to find another elastic explorer if needed :)

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "db:use": "node ./scripts/db-use.js",
     "elastic:activate": "node ./scripts/elastic.js activate",
     "elastic:deactivate": "node ./scripts/elastic.js deactivate",
+    "elastic:hq": "node ./scripts/elastic.js hq",
```
Comment (sagarsdeshmukh) [inline package.json:33]: As we remove elastic hq, we need to remove this line as well. right?

Comment (lithrel) [reply package.json:33]: :grimacing: good point !

**Review by sagarsdeshmukh — APPROVED** [review]
Tested on local, PR looks good to me 👍

**Pull request merged**